### ISSUE TITLE
Cannot find module 'cross-spawn' when running via npx

### ISSUE THREAD

**Issue by jnv** [codeceptjs/CodeceptJS#2729]
#### What are you trying to achieve?

I am trying to fix [mock-request](https://github.com/codeceptjs/mock-request) package and get it up to speed with CodeceptJS v3. You can find my fork here: https://github.com/jnv/mock-request

I have updated the spec syntax and dependencies, but I can't seem to get it running.

#### What do you get instead?

When running codeceptjs via `npx` I get the following error:

```
$ npx codeceptjs run -c test/codecept.puppeteer.conf.js --debug --grep @Puppeteer                                                                                         
Cannot find module 'cross-spawn'
Require stack:
- .../mock-request/node_modules/codeceptjs/lib/command/init.js
- .../node_modules/codeceptjs/bin/codecept.js
```

You can see the failure in GitHub Action: https://github.com/jnv/mock-request/runs/1589145328?check_suite_focus=true#step:8:4

### Details

* CodeceptJS version: 3.0.3
* NodeJS Version: v12 and v14
* Operating System: Linux, macOS
* puppeteer || webdriverio
* Configuration file:

Puppeteer:

```js
module.exports.config = {
  tests: './*_test.js',
  timeout: 10000,
  output: './output',
  helpers: {
    Puppeteer: {
      url: 'http://0.0.0.0:8000',
      show: false,
      chrome: {
        args: [
          '--disable-web-security',
          '--no-sandbox',
          '--disable-setuid-sandbox',
        ],
      },
    },
    FileSystem: {},
    MockRequestHelper: {
      require: '../index.js'
    },
  },
  include: {},
  bootstrap: false,
  mocha: {},
  name: 'acceptance',
};
```

WebDriver:

```js
module.exports.config = {
  tests: './*_test.js',
  timeout: 10000,
  output: './output',
  helpers: {
    WebDriver: {
      url: 'http://127.0.0.1:8000',
      browser: 'chrome',
      // host: TestHelper.seleniumHost(),
      // port: TestHelper.seleniumPort(),
      // disableScreenshots: true,
      // desiredCapabilities: {
      //   chromeOptions: {
      //     args: ['--headless', '--disable-gpu', '--window-size=1280,1024'],
      //   },
      // },
    },
    MockRequestHelper: {
      require: '../index.js'
    },
  },
  include: {},
  bootstrap: done => setTimeout(done, 5000), // let's wait for selenium
  mocha: {},
  name: 'acceptance',
};
```

#### Notes

The issue seems to be a missing runtime dependency on cross-spawn. It is required here:

https://github.com/codeceptjs/CodeceptJS/blob/0e48818d9bf010ae15fedcda411dd871953c1802/lib/command/init.js#L7

But it seems to be missing in package.json:

https://github.com/codeceptjs/CodeceptJS/blob/0e48818d9bf010ae15fedcda411dd871953c1802/package.json#L56-L88

Simplest workaround I use for now is to add cross-spawn dependency directly to the project (as devDependency in this case).

**Comment by vipulgupta2048** [comment]
Can confirm the recently released version 3.0.3 breaks `npx codecept init` script. Way too much time wasted trying to locate this dependency. Hope this helps ^^ someone. If folks are not able to make it work, try running with `"codeceptjs": "3.0.2"` instead in your devDependencies.

**Comment by vipulgupta2048** [comment]
Made a quick PR and added it here ^
